engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/callcaching/CallCacheHashingJobActor.scala

```diff
@@ -233,9 +234,9 @@ object CallCacheHashingJobActor {
 
   object CallCacheHashingJobActorData {
     // Slick will eventually build a prepared statement with that many parameters. Don't set this too high or it will stackoverflow.
-    val BatchSize = 100
+    val BatchSize: Int = ConfigFactory.load().getInt("system.file-hash-batch-size")
```

Comment:
ToL: Calling `ConfigFactory.load()` is suboptimal but works.

- Harder to test different values since the config value is read as a singleton
- The config objects are definitely available elsewhere in the code but do not seem to be plumbed down to this level of code. I suspect usually folks have plumbed down only the extracted values and not sent in the `Config` object itself